GeoLib/AnalyticalGeometry.h

```diff
@@ -60,11 +60,28 @@ Orientation getOrientation (const GeoLib::Point* p0,
  * @param plane_normal the normal of the plane the polygon is located in
  * @param d parameter from the plane equation
  */
-void getNewellPlane (const std::vector<GeoLib::Point*>& pnts,
+template <class T_POINT>
+void getNewellPlane (const std::vector<T_POINT*>& pnts,
```

Comment:
@TomFischer Who is "Newell"?
Sarcasm aside, from the documentation it is not clear what is modified and how.